File: ATphp/sms.php
<?php
$headers = [
    'apiKey: 4b04974557ed796e6978f3af009758e104d6284be7bd5e657eba9c8d443043b2',
    'Content-Type: application/json',
    'Accept: application/json',
];

require 'vendor/autoload.php';
require_once 'constants.php';
use AfricasTalking\SDK\AfricasTalking;

// Set your app credentials
$username   = "sandbox";
$apiKey     = "4b04974557ed796e6978f3af009758e104d6284be7bd5e657eba9c8d443043b2";

// Initialize the SDK
$AT         = new AfricasTalking($username, $apiKey);

// Get the SMS service
$sms        = $AT->sms();

// Set the numbers you want to send to in international format
$recipients = "+254794824427";

// Set your message
$message    = "I'm a lumberjack and its ok, I sleep all night and I work all day";

// Set your shortCode or senderId
$from       = "7309";

try {
    // Thats it, hit send and we'll take care of the rest
    $result = $sms->send([
        'to'      => $recipients,
        'message' => $message,
        'from'    => $from
    ]);

    print_r($result);
} catch (Exception $e) {
    echo "Error: ".$e->getMessage();
}

File: sms.py
# works with both python 2 and 3
from __future__ import print_function

import africastalking

class SMS:
    def __init__(self):
        # Set your app credentials
        self.username = "soroel"
        self.api_key = "1a5a7e4c619fb59d7f5d4a32f72a1cbd3ef47675b010032229c279b0ce71c9d1"

        # Initialize the SDK
        africastalking.initialize(self.username, self.api_key)

        # Get the SMS service
        self.sms = africastalking.SMS

    def send(self):
        # Set the numbers you want to send to in international format
        recipients = ["+254794846211"]

        # Set your message
        message = "heeeey, this is a test message from miss developer"

        # Set your shortCode or senderId
        #sender = "soroel"
        try:
            # Thats it, hit send and we'll take care of the rest.
            response = self.sms.send(message, recipients)
            print (response)
        except Exception as e:
            print ('Encountered an error while sending: %s' % str(e))

if __name__ == '__main__':
    SMS().send()